includes/wikia/Wikia.php

```diff
@@ -166,6 +167,25 @@ public static function invalidateFavicon() {
 	}
 
 	/**
+	 * Return either a path to locally-uploaded Wiki.png file or a shared file taken from the code repo
+	 * (if there's no Wiki.png file uploaded on a wiki)
+	 *
+	 * @see SUS-1165
+	 *
+	 * @return string
+	 */
+	public static function getWikiLogoFullUrl() {
```

Comment:
What if `$wgLogo` is set up to a different path than `wiki.png`? That wouldn't show the selected icon and fallback to `skins/common/images/wiki.png`. Is it OK? Are there such pages?